Nodejs/Product/ProjectWizard/NodejsPackageParametersExtension.cs

```diff
@@ -19,7 +19,7 @@
 using System.Text.RegularExpressions;
 using EnvDTE;
 using Microsoft.VisualStudio.TemplateWizard;
-

```

Comment:
I've tried but I can't undo this change. It's whitespace only but git refuses to see that anything has been changed even after checking out this file directly from this repo directly.